### PR TITLE
cod-mahlbema-3

### PR DESCRIPTION
**Pull request by mike-allaway**
This branch includes work done by Mike Allaway of the RSG at University of Birmingham, for the cod-mahlbema-3 coding engagement.

Its aim is to tidy up the code, improve the testing, and update documentation accordingly.

### CODE REVIEW

**Review by mike-allaway — COMMENTED** [review]
I'm happy with this @edmondac

File: R/corp_coco.R [hunk 1]
```diff
@@ -33,83 +33,83 @@ corp_get_metadata.corp_coco <- function(obj) {list(
...
 plot.corp_coco <- function(x, as_matrix = FALSE, nodes = NULL, forest_plot_args = NULL, ...) {
     # hack to stop R CMD check warnings
-    effect_size = V1 = y = CI_upper = CI_lower = NULL
+    effect_size <- V1 <- y <- CI_upper <- CI_lower <- NULL
 
-    # can't use metadata attr - data.table functions do not preserve non-data.table attributes - https://github.com/Rdatatable/data.table/issues/995
-    if(! is.null(nodes)) {
-        if(! all(nodes %in% unique(x[,x]))) {
-            warning('Some of the supplied plot nodes are not in the significant results and therefore will be absent from the plot')
+    # can"t use metadata attr - data.table functions do not preserve non-data.table attributes - https://github.com/Rdatatable/data.table/issues/995
```
Comment (edmondac) [inline R/corp_coco.R]:
```suggestion
    # can't use metadata attr - data.table functions do not preserve non-data.table attributes - https://github.com/Rdatatable/data.table/issues/995
```

File: ChangeLog [hunk 2]
```diff
@@ -1,3 +1,10 @@
+2021-04-29  Mike Allaway  <m.j.allaway@bham.ac.uk>
```
Comment (edmondac) [inline ChangeLog]:
```suggestion
2021-04-29  Mike Allaway <m.j.allaway@bham.ac.uk> and Andrew Edmondson <a.c.edmondson@bham.ac.uk>
```

**Review by edmondac — APPROVED** [review]
I'm happy with this now

**Pull request merged**